lib/pymedphys/docs/README.rst

```diff
@@ -92,7 +92,7 @@ For Python Users
   - Start here if you are fluent in Python and just want a quick instruction on how
     to get started.
 
-- `Tutorials`_ **[Don't yet exist]**
+- `Tutorials`_
```

Comment:
This is a little cheeky, yes. But... I want the notice linked to, to be able to be read...